be/src/agent/task_worker_pool.cpp

```diff
@@ -132,13 +133,15 @@ void finish_task(const TFinishTaskRequest& finish_task_request) {
     constexpr int TASK_FINISH_MAX_RETRY = 3;
     while (try_time < TASK_FINISH_MAX_RETRY) {
         DorisMetrics::instance()->finish_task_requests_total->increment(1);
+        DorisBvarMetrics::instance()->finish_task_requests_total->increment(1);
```

Comment:
consider inherit from the original metrics and rewrite the `increment()` to reduce numbers of files modified. e.g.
```c++
class BvarCounter : public IntCounter {
void increment(int64_t) { ... }
}
```

Reply:
To be precise, the original metric framework was outdated and had poor performance. This PR only utilized the fast counter feature of the bvar class to replace all the original metrics